phoenix-pherf/src/test/java/org/apache/phoenix/pherf/RuleGeneratorTest.java

```diff
@@ -175,7 +175,26 @@ public void testSequentialDataSequence() throws Exception {
         Column targetColumn = null;
         for (Column column : model.getDataMappingColumns()) {
             DataSequence sequence = column.getDataSequence();
-            if (sequence == DataSequence.SEQUENTIAL) {
+            if (!DataTypeMapping.INTEGER.equals(column.getType()) && sequence == DataSequence.SEQUENTIAL) {
+                targetColumn = column;
+                break;
+            }
+        }
+        assertNotNull("Could not find a DataSequence.SEQENTIAL rule.", targetColumn);
+        assertMultiThreadedIncrementValue(targetColumn, rulesApplier);
+    }
+
+    @Test
+    public void :testSequentialIntegerDataSequence() throws Exception {
```

Comment:
extra ":" , this won't compile.

Reply:
Think this was accidental after realizing i had another commit on this branch during cleanup.  I'll fix and submit hadoopQA.